draft-ietf-quic-http.md

```diff
@@ -1256,7 +1261,8 @@ Each frame's payload MUST contain exactly the fields identified in its
 description.  A frame payload that contains additional bytes after the
 identified fields or a frame payload that terminates before the end of the
 identified fields MUST be treated as a connection error of type
-H3_FRAME_ERROR; see {{errors}}.
+H3_FRAME_ERROR; see {{errors}}.  In particular, redundant length encodings MUST
+be verified to be self-consistent.
```

Comment:
We do call this out in Section 10.8 already, so I don't mind if we skip this.  It might be worth reiterating, though, given how many vulnerabilities this sort of thing has caused.

Reply:
If this is a forward reference, then include that reference;

```suggestion
be verified to be self-consistent; see {{frame-parsing}}.
```

Reply:
Good call; I wrote this before I got that far in the doc to know there was a forward reference to make.